### ISSUE TITLE
Add customs fonts per language

### ISSUE THREAD

**Issue by M-Biggles** [jzohrab/lute-v3#278]
A very useful feature would be to add a 'Font' field (new/edit language pages).

Many languages share the same script but can be quite distinct typographically.

**Comment by jzohrab** [comment]
Hi @M-Biggles - can you give me a concrete example?  I can't understand what this request is about.  Is Font like "helvetica", or something totally different?  Is this related to #220 , "writing systems"?

**Comment by M-Biggles** [comment]
In Custom Styles, we can set fonts like this:

span.textitem[data-lang-id="3"]{ font-family:"David CLM"; }
span.textitem[data-lang-id="34"]{ font-family:"Akkadian"; }
span.textitem[data-lang-id="57"]{ font-family:"Samyak Tamil"; }
span.textitem[data-lang-id="58"]{ font-family:"Noto Sans"; }
span.textitem[data-lang-id="62"]{ font-family:"Tiro Devanagari Marathi"; }

It's possible now but difficult (especially for non-coders). With fonts part of language presets, we could offer better language support. This isn't a real issue for something like English or Spanish, but for many languages, the difference in font is a big deal.

**Comment by GrimPixel** [comment]
Even for the same language, different regions have different glyph standards. The “language” would be actually “language-script-region”.

**Comment by M-Biggles** [comment]
> Even for the same language, different regions have different glyph standards. The “language” would be actually “language-script-region”.

Yeah, absolutely. That's why I decided to make this issue; it has a big impact because it really helps to match the look and feel of the language to the reality.

**Comment by GrimPixel** [comment]
The same text can be assigned more than one “language-script-region” value, like `deu-Latn-DE` and `deu-Latf-DE`, where the latter one is in Fraktur type.

**Comment by M-Biggles** [comment]
> The same text can be assigned more than one “language-script-region” value, like `deu-Latn-DE` and `deu-Latf-DE`, where the latter one is in Fraktur type.

Yeah, for sure. I've had the same idea in mind. Per-text settings would be nice, but just getting per-language settings is our first step. Definitely worth returning to it when the time is right.
